sphinx/development/doctests.rst

```diff
@@ -3,20 +3,121 @@ Doctests guide
 
 (Work in progress)
```

Comment:
@jdbancal This describes the new doctest syntax. I'm now adapting the code to it!

Reply:
Yep, sounds great!!

Regarding the explanation itself in this document, I tried to clarify slightly some points, please check if I understood everything correctly.

Also one point which was confusing to me was the usage of double quotes, e.g. in
```
>>> "command-text" [...] [% ["comment-text"] [doctest: "flags"]]
```
We see from the examples that we actually should not use quotes, as in we don't mean to write
```
>>> "1+1"
```
so I was wondering if we could just use
```
>>> command-text [...] [% [comment-text] [doctest: flags]]
```
Otherwise, maybe we can just add a short comment for the uninformed reader so that it's clear that the quotes are here to denote strings, and they should not be typed per se.

Reply:
Good idea! corrected